airflow/config_templates/config.yml

```diff
@@ -517,6 +517,17 @@
       type: string
       example: ~
       default: "task"
+    - name: donot_modify_handlers
+      description: |
+        If ``donot_modify_handlers = True``, we do not modify logging handlers in task_run command.
+        If the flag is set to False, we remove all handlers from the root logger.
+        and add all handlers from 'airflow.task' logger to the root Logger. This is done
+        to get all the logs from the print & log statements in the DAG files before a task is run
+        The handlers are restored after the task completes execution.
+      version_added: ~
+      type: boolean
+      example: ~
+      default: "False"
```

Comment:
I think this should be a hidden setting -- i.e. not actually documented, nor appearing in default_airflow.cfg

Reply:
Updated

Reply:
in https://github.com/apache/airflow/pull/9363/commits/6f8225fa5f5cd84cd94c40aefef736b8328b6529